source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -82,6 +82,7 @@ export class CardElementPeerRegistry extends DesignerPeerRegistry<CardElementTyp
         this.registerPeer(Adaptive.FactSet, DesignerPeers.FactSetPeer, DesignerPeerCategory.Containers, "acd-icon-factSet");
 
         this.registerPeer(Adaptive.TextBlock, DesignerPeers.TextBlockPeer, DesignerPeerCategory.Elements, "acd-icon-textBlock");
+        this.registerPeer(Adaptive.RichTextBlock, DesignerPeers.RichTextBlockPeer, DesignerPeerCategory.Elements, "acd-icon-richTextBlock");
```

Comment:
Cool so you actually are including designer support for richtextblock? @andrewleader, is this what we were talking about earlier in the am wrt tooling complexities with richtextblock?

Reply:
That looks like that's just adding it to the list of controls. When we say "designer support", we mean support for editing the text contents. I believe the experience will be that you can drag a blank one in, but to add any text to it, you have to go to the JSON editor at the bottom, correct me if I'm wrong David!

Reply:
This is baseline/minimum designer support. E.g. you can drag/drop a RichTextBlock and edit its core properties, but you can't edit the runs easily, you have to do it in the JSON editor.

Reply:
Cool thanks! 

---
In reply to: [276842538](https://github.com/Microsoft/AdaptiveCards/pull/2714#discussion_r276842538) [](ancestors = 276842538)